pkg/controller.v1/common/job.go

```diff
@@ -191,7 +193,7 @@ func (jc *JobController) ReconcileJobs(
 			return err
 		}
 
-		if jc.Config.EnableGangScheduling {
+		if jc.Config.EnableGangScheduling() {
```

Comment:
Is this typo or intentional?

Reply:
This is intentional. Because this PR is expanding the gang-scheduler options, instead of adding another field for the name of gang-scheduler, `Config` only stores the name of gang-scheduler. The `EnableGangScheduling()` method returns false if `None` is set for the name of the gang-scheduler, otherwise it returns true.